frontend/src/Explorer/View/Dashboard/Blocks.purs

```diff
@@ -107,12 +112,38 @@ blockRow state (CBlockEntry entry) =
     P.link (toUrl <<< Block $ entry ^. cbeBlkHash)
         [ P.className "blocks-body__row" ]
         [ blockColumn <<< show $ entry ^. cbeHeight
-        , blockColumn <<< show <<< unwrap $ entry ^. (cbeTimeIssued <<< _NominalDiffTime)
+        , blockColumn (prettyDuration language (Milliseconds $ unwrap $ entry ^. (cbeTimeIssued <<< _NominalDiffTime)))
+        -- , blockColumn <<< show <<<  $ unwrap $ entry ^. (cbeTimeIssued <<< _NominalDiffTime)
         , blockColumn <<< show $ entry ^. cbeTxNum
         , blockColumn <<< show $ entry ^. (cbeTotalSent <<< _Coin <<< getCoin)
-        , blockColumn <<< fromMaybe (translate (I18nL.common <<< I18nL.cUnknown) $ state ^. lang) $ entry ^. cbeRelayedBy
+        , blockColumn <<< fromMaybe (translate (I18nL.common <<< I18nL.cUnknown) language) $ entry ^. cbeRelayedBy
         , blockColumn <<< show $ entry ^. cbeSize
         ]
+    where
+      language      = state ^. lang
+
+-- cDays cHours cMinutes cSeconds
+prettyDuration :: forall a. Duration a => Language -> a -> String
+prettyDuration lang dur | convertDuration dur < Minutes 1.0 = "< 1 " <> translate (I18nL.common <<< I18nL.cMinutes) lang
+                        | otherwise = trim $ joinWith " " $ map (uncurry showIfNonZero)
+                             [ Tuple d translationDays
+                             , Tuple h translationHours
+                             , Tuple m translationMinutes
+                             ]
+  where
+    translationMinutes  = translate (I18nL.common <<< I18nL.cMinutes) lang
+    translationHours    = translate (I18nL.common <<< I18nL.cHours) lang
+    translationDays     = translate (I18nL.common <<< I18nL.cDays) lang
+    m = floor <<< unMinutes $ convertDuration dur `sub` convertDuration (Days $ toNumber d) `sub` convertDuration (Hours $ toNumber h)
+    h = floor <<< unHours $ convertDuration dur `sub` convertDuration (Days $ toNumber d)
+    d = floor <<< unDays $ convertDuration dur
+    unHours     (Hours m) = m
```

Comment:
You could use `unwrap` https://pursuit.purescript.org/packages/purescript-newtype/1.3.0/docs/Data.Newtype#v:unwrap for all newtypes (`Hours`, `Days`, `Minutes`) instead of using custom helper functions `unHours`, `unDays` and `unMinutes`,